nomad/grapl-core.nomad

```diff
@@ -641,60 +634,6 @@ job "grapl-core" {
 
     service {
       name = "graph-merger"
-      //      connect {
-      //        sidecar_service {
-      //          proxy {
-      //            dynamic "upstreams" {
-      //              iterator = alpha
-      //              for_each = local.dgraph_alphas
-      //
-      //              content {
-      //                destination_name = "dgraph-alpha-${alpha.value.id}-grpc-public"
-      //                local_bind_port  = alpha.value.grpc_public_port
-      //              }
-      //            }
-      //          }
-      //        }
-      //      }
-    }
-  }
-
-  group "provisioner" {
```

Comment:
Provisioner has been moved to `nomad/grapl-provisioner.nomad` so we can make it "depend" on a complete dgraph deployment.